Common/include/Luau/Bytecode.h

```diff
@@ -426,7 +426,7 @@ enum LuauBytecodeTag
 {
     // Bytecode version; runtime supports [MIN, MAX], compiler emits TARGET by default but may emit a higher version when flags are enabled
     LBC_VERSION_MIN = 3,
-    LBC_VERSION_MAX = 4,
+    LBC_VERSION_MAX = 5,
```

Comment:
We should also add v5 to a comment describing bytecode versions closer to the top of the file